edu-services/gradebook-service/hibernate/src/java/org/sakaiproject/tool/gradebook/CourseGradeRecord.java

```diff
@@ -33,6 +33,7 @@
 	private static final long serialVersionUID = 1L;
 
 	private String enteredGrade;
+	private Double pointsEarned;
```

Comment:
@Setter and @Getter

Reply:
get method is already created and we have modify it, that's the reason we didn't add a getter and setter method.